noiseworks/settings.py

```diff
@@ -273,3 +273,31 @@
 file_storage_relative_path = env.str("FILE_STORAGE_RELATIVE_PATH", None)
 if file_storage_relative_path:
     MEDIA_ROOT = BASE_DIR / file_storage_relative_path
+
+# Logging
+
+LOGGING = {
+    "version": 1,
+    "disable_existing_loggers": False,
+    "formatters": {
+        "timestamp": {
+            "format": "{asctime} {levelname} {message}",
+            "style": "{",
+        },
+    },
+    "handlers": {
+        "file": {
+            "level": env("LOG_LEVEL"),
```

Comment:
Should this specify a default here? I believe it'll default to `WARNING` if `level` is `None`, but might be better to be explicit rather than implicit.